pkg/fileutils/fileutils.go

```diff
@@ -105,33 +108,33 @@ func CopyFile(src string, dst string) (err error) {
 		d *os.File
 	)
 	if !IsRegularFile(src) {
-		return fmt.Errorf("copy file:%s error, is not a regular file", src)
+		return fmt.Errorf("failed to copy %s to %s: src is not a regular file", src, dst)
 	}
 	if s, err = os.Open(src); err != nil {
-		return err
+		return fmt.Errorf("failed to copy %s to %s when opening src file: %v", src, dst, err)
 	}
 	defer s.Close()
 
 	if PathExist(dst) {
-		return fmt.Errorf("copy file:%s error, dst file already exists", dst)
+		return fmt.Errorf("failed to copy %s to %s: dst file already exists", src, dst)
 	}
 
 	if d, err = OpenFile(dst, os.O_RDWR|os.O_TRUNC|os.O_CREATE, 0755); err != nil {
-		return err
+		return fmt.Errorf("failed to copy %s to %s when opening dst file: %v", src, dst, err)
```

Comment:
Please make this distinguished from the Line 114 of this file, since when log shows the error message, you will be confused whether it is from L114 or L123.
I am wondering if I am right. Thanks.

Reply:
L114: "failed to copy %s to %s when opening *src* file: %v"
L114: "failed to copy %s to %s when opening *dst* file: %v"
These two strings are the same length so that they cannot be easily distinguished, I will modify src to source and dst to destination.
Thanks for your comment.

Reply:
Done.